pkg/webhook/webhook.go

```diff
@@ -232,6 +232,7 @@ func (wh *webhook) validatePVCCreateRequest(req *v1beta1.AdmissionRequest) *v1be
 	// get the snapshot object to get snapshotdata object
 	snapObj, err := wh.snapClientSet.OpenebsV1alpha1().VolumeSnapshots(pvc.Namespace).Get(snapname, metav1.GetOptions{})
```

Comment:
if snapname doesn't exists, does this call returns error? Please verify

Reply:
Yes, it will return an error if it is not able to get the data.
```
Error from server: error when creating "snapshot_claim.yaml": admission webhook "admission-webhook.openebs.io" denied the request: Failed to get the snapshot object for snapshot name: 'snapshot-demo-cstor1' namespace: 'default' error: 'volumesnapshots.volumesnapshot.external-storage.k8s.io "snapshot-demo-cstor1" not found'
```